bool stat
---
int64 goal_num
int64 cancel_num